lib/bigfiles/quality_config.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require_relative 'quality_threshold'
+
+module BigFiles
+  # TODO: This should live in quality gem
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker